discovery_engine_core/web-api/src/migrations/20220926145400_document.sql

```diff
@@ -0,0 +1,24 @@
+--  Copyright 2022 Xayn AG
+--
+--  This program is free software: you can redistribute it and/or modify
+--  it under the terms of the GNU Affero General Public License as
+--  published by the Free Software Foundation, version 3.
+--
+--  This program is distributed in the hope that it will be useful,
+--  but WITHOUT ANY WARRANTY; without even the implied warranty of
+--  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+--  GNU Affero General Public License for more details.
+--
+--  You should have received a copy of the GNU Affero General Public License
+--  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+CREATE TABLE IF NOT EXISTS document (
+    doc_id TEXT NOT NULL,
+    user_id TEXT NOT NULL,
+    last_view TIMESTAMPTZ NOT NULL DEFAULT Now(),
+    user_reaction SMALLINT NOT NULL DEFAULT 0,
```

Comment:
I wasn't aware that we have `UserReaction` enum, so we should create a ticket for using that too in `center_of_interest` table instead of a boolean, wdyt @acrrd?

Reply:
On elastic search? Yes please. Actually we need to define a new `UserReaction` in the web-api because the one we have has `neutral` that we don't want to expose at the moment.

@janpetschexain this also means that we cannot have a default for it.

Reply:
We want `doc_id, user_id, timestamp` as the primary key otherwise we lose some interaction.

Reply:
>On elastic search? Yes please.
no, I was referring to a `center_of_interest` Postgres table, where we store kind of interaction as a boolean

Reply:
we could also use a boolean here for now and decide for some enum later on

Reply:
ah sorry. There matters less. A coi can only be positive or negative I don't see much room for extensions there.

Reply:
is postgres is storing boolean as a `0` and `1` internally and we can avoid migration if we change the type then this could work too.. right now we're not in production so it would be better to use a type that is more future-proof

Reply:
> A coi can only be positive or negative

ah, right, good point

Reply:
- added `time_stamp` to the PK
- added `UserReaction` wrapper & removed default for `user_reaction`